interfaces/policy/policy.go

```diff
@@ -155,6 +160,14 @@ func (connc *ConnectCandidate) SlotAttr(arg string) (any, error) {
 	return nestedGet("slot", connc.Slot, arg)
 }
 
+func (connc *ConnectCandidate) SlotCompatAttr(arg string) (any, error) {
+	if !connc.CompatEnabled {
+		logger.Debugf("compatibility labels are disabled")
+		return nil, errors.New("compatibility labels are disabled")
```

Comment:
how does this surfaces in practice?

Reply:
those are kind of expected historically to swallow errors, as they in the end are only a true/false thing